client/src/components/Create.js

```diff
@@ -0,0 +1,130 @@
+import React from 'react'
+import { SegmentedControl } from 'segmented-control'
+import styled from 'styled-components/macro'
+
+export default function Create() {
+  return (
+    <ContentWrapper>
+      <h2>Level up!</h2>
+      <p>Add new games to your library.</p>
+      <form>
+        <label for="title">
+          Title:{' '}
```

Comment:
Did Prettier add the {' '} or does it have a purpose?

Reply:
Prettier did :D